app/assets/stylesheets/modules/mobile-nav.scss

```diff
@@ -15,6 +15,12 @@
   z-index: 98;
   transition: all 0.5s;
   opacity: 0;
+  overflow-y: scroll;
+  > div {
```

Comment:
What is this padding for?

Reply:
without this padding it won't show the coinfi logo on shorter heights, like mobile landscape